hudi-sync/hudi-hive-sync/src/test/java/org/apache/hudi/hive/testutils/HiveTestService.java

```diff
@@ -215,8 +215,9 @@ private boolean waitForServerUp(HiveConf serverConf, String hostname, int timeou
     while (true) {
       try {
         new HiveMetaStoreClient(serverConf);
+        Thread.sleep(30000);
```

Comment:
Ran into issues here with some hive tests being flaky and increased sleep here. Similar as my above comment in `ITTestBase`